src/avram/queryable.cr

```diff
@@ -36,9 +36,19 @@ module Avram::Queryable(T)
       new.last?
     end
 
-    def self.truncate
+    # Removes all data from a table using the TRUNCATE postgres SQL command.
+    #
+    # You should not run this command without `cascade: true` if your table
+    # columns are referenced by other foreign key constraints. Use *delete*
+    # instead if you don't want to accidentally delete rows referenced
+    # elsewhere.
+    #
+    # To delete all data referenced by foreign keys as well, set *cascade*
+    # to true.
+    def self.truncate(cascade : Bool = false)
```

Comment:
```suggestion
    def self.truncate(*, cascade : Bool = false)
```

With this, it enforces that a named arg is always used. This will prevent someone using `Thing.truncate(true)` in their codebase.